### ISSUE TITLE
When QP & ST are on Log page doesn't activate in case of syntax error.

### ISSUE THREAD

**Issue by SharmaAntriksh** [DaxStudio/DaxStudio#1187]
Inside DS when viewing Results pane and:

When Query Plan and Server Timings are on and you run a query succesfully and in the next try you change the code and run it and it results in a syntax error then DAX Studio should switch to Log pane, currently it just repeats the result of previous execution and the user thinks code is working, which is causing issue because I kept thinking the code is correct 🤣 but minutes later I found out the results aren't changing. 

```
EVALUATE
	ADDCOLUMNS (
		VALUES ( Products[Brand] ),
		"Avg Sales", [Sales Amount] / CALCULATE ( COUNTROWS ( Products ) ),
		-- First run this ^
 		"Avg Sales 2", AVERAGEX ( 
 			CALCULATETABLE ( Products ), 
 			CALCULATE ( [Sales Amount] ), 
 			REMOVEFILTERS ( Products ) 
 		)
 		-- Then this ^
	)

```
This doesn't happen when QP and ST are off

**Comment by dgosbell** [comment]
Thanks for logging this. You are correct that this behaviour is a bit confusing. I did set things up so that when you have a trace running I don't switch panes, but this was only meant to prevent switching to the results when you were running a trace, it probably should still switch to the log if there is an error. I'm also thinking I should clear any previous results at the start of a query to avoid this sort of confusion.

**Comment by SharmaAntriksh** [comment]
Yup, clearing the Results pane plus the QP/ST tabs do show the error in a Yellow bar, maybe that can be just replicated to the results pane as well.
![image](https://github.com/DaxStudio/DaxStudio/assets/73054674/79930ac9-94b4-4d24-8988-1c840f287641)
